packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -16,7 +16,39 @@ import ReachabilityRequest from './request';
 const DEFAULT_TIMEOUT = 3000;
 const VIDEO_MESH_TIMEOUT = 1000;
 
-export type ICECandidateResult = {clusterId: string; elapsed?: string | null; publicIPs?: string[]};
+// result for a specific transport protocol (like udp or tcp)
+export type TransportResult = {
+  reachable: 'true' | 'false';
```

Comment:
any reason we're using string literals and not `boolean`?

Reply:
This is existing code, I only added the type for it, I guess it's like that because that's what the server expects. I don't want to risk changing it now. I mean I could try, but then I need to join a test meeting and confirm in Orpheus logs that it reads it correctly and takes it into account correctly when deciding call placement, so it will take time.

Reply:
nah, it's ok i don't mind, i thought there is a reason for it